server.go

```diff
@@ -1699,6 +1703,33 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	return s, nil
 }
 
+// UpdateRoutingConfig is a callback function to update the routing config
+// values in the main cfg when the lncli setmccfg is called.
```

Comment:
nit: it will be called on `SetMissionControlConfig` which is not limited to `lncli setmcfg`